community/modules/scheduler/k8s-cluster/variables.tf

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+variable "project_id" {
+  description = "The project ID to host the cluster in."
+  type        = string
+}
+
+variable "name_suffix" {
+  description = "Custom cluster name postpended to the `deployment_name`. See `prefix_with_deployment_name`."
+  type        = string
+  default     = ""
+}
+
+variable "deployment_name" {
+  description = "Name of the HPC deployment. Used in the GKE cluster name by default and can be configured with `prefix_with_deployment_name`."
+  type        = string
+}
+
+variable "prefix_with_deployment_name" {
+  description = "If true, cluster name will be prefixed by `deployment_name` (ex: <deployment_name>-<name_suffix>)."
+  type        = bool
+  default     = true
+}
+
+variable "region" {
+  description = "The region to host the cluster in."
+  type        = string
```

Comment:
when you specify the region, does it always pick three zones out of this region? how do you make sure it is the right three? maybe that will be a more important decision at the node pool level...